src/common/extract_scores/script.R

```diff
@@ -17,14 +17,20 @@ scores <- map_df(par$input, function(inp) {
   cat("Reading '", inp, "'\n", sep = "")
   ad <- read_h5ad(inp)
 
-  for (uns_name in c("dataset_id", "method_id", "metric_id", "metric_value")) {
```

Comment:
This makes sense, but maybe it's better to make this a parameter of the component. That is, something like this.

```yaml
    - name: "--column_names"
      type: "string"
      multiple: true
      default: [ dataset_id, method_id, metric_id, metric_value ]
      description: "Which fields from adata.uns to extract and store as a data frame."
```

You can then use `par$column_names` instead of `uns_names`.